src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Instance/Authenticator.cs

```diff
@@ -107,6 +108,7 @@ public async Task UpdateFromTemplateAsync(RequestContext requestContext)
             {
                 var authorityUri = new Uri(Authority);
                 var host = authorityUri.Host;
+                var authority = authorityUri.Authority;
```

Comment:
Seems like authority is not used any more?

Reply:
Yes. will remove